### PR TITLE
Setup CD

### LINKED ISSUE #5: Set up the CD
We need an automated pipeline for deployment purposes.
This involves a production environment to be set.

### PR DESCRIPTION
**Pull request by AnastasiaPromtep**
Closes #5

### CODE REVIEW

File: .github/workflows/deploy-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          repository: lapoissonnerie/touiteur-dev
+          tags: "latest,dev"
+
+      - uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.DEPLOY_HOST }}
+          username: ${{ secrets.DEPLOY_USERNAME }}
+          key: ${{ secrets.DEPLOY_SSHKEY }}
+          script: |
+            wget https://raw.githubusercontent.com/lapoissonnerie/touiteur-front/develop/stack-dev.yaml -O /data/touiteur/api/stack-dev.yaml
```
Comment (AnastasiaPromtep) [inline .github/workflows/deploy-dev.yaml:29]: While I guess what it does, I wonder how it works. Is it the Github API ?

**Comment by AnastasiaPromtep** [comment]
I think those keys need to be set up on the VPS:
- secrets.DEPLOY_HOST
- secrets.DEPLOY_USERNAME
- secrets.DEPLOY_SSHKEY

File: .github/workflows/deploy-dev.yaml (hunk 1 above)
Comment (maximelouet) [reply .github/workflows/deploy-dev.yaml:29]: That's the path of the raw file (accessible from the `Raw` button on a file) dumb slut

**Comment by maximelouet** [comment]
Done :ok_hand:

**Pull request merged**